ui/src/app/pages/homepage/registration-dashboard.spec.tsx

```diff
@@ -15,8 +17,13 @@ describe('RegistrationDashboard', () => {
     (<RegistrationDashboard {...props}/>);
   };
 
-  beforeEach(() => {
+  beforeEach(async() => {
     registerApiClient(ProfileApi, new ProfileApiStub());
+    userProfileStore.next({
```

Comment:
Side note: this seems like a pretty clunky way to get the stubbed profile into the profile store, but I didn't see any obviously better pattern. Open to suggestions.

Reply:
Looks like that's the standard pattern, agreed it's a bit clunky.